4_evaluate/src/write_metrics.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+
+import sys
+sys.path.insert(1, '4_evaluate/src')
+sys.path.insert(1, '5_visualize/src')
+from plot_metrics import load_predictions, rms
+
+"""
+Write evaluation metrics for one model to a csv file. This will be a single row in a csv.
+"""
+
+
+def compute_metrics(predictions_filepath):
+    """
+    Compute metrics for a dataset, including overall RMSE, median and mean RMSE
+    by lake, the total number of observations, and the number of lakes.
+
+    :param predictions_filepath: Path to csv with model predictions
+    :returns: DataFrame of metrics computed for the predictions
+
+    """
+    preds = load_predictions(predictions_filepath)
+    rmse_lake = preds.groupby('site_id')['residual'].aggregate(rms).rename('LSTM_rmse')
```

Comment:
Is the `.rename('LSTM_rmse')` doing anything?

It seems that that is more of a `pandas`-column-renaming thing, but `rmse_lake` is being used more-or-less as an array which doesn't need a name.

Reply:
You're right - I think that's a leftover from code that was using the `pandas` column names directly for plotting.
```suggestion
    rmse_lake = preds.groupby('site_id')['residual'].aggregate(rms)
```